### PR TITLE
MPLS support

### PR DESCRIPTION
**Pull request by ishidawataru**

### CODE REVIEW

File: route_linux.go [hunk 1]
```diff
@@ -60,6 +61,74 @@ func (n *NexthopInfo) ListFlags() []string {
...
 
+type MPLSDestination struct {
+	Labels []int
+}
+
+func (d *MPLSDestination) Family() int {
+	return nl.FAMILY_MPLS
+}
+
+func (d *MPLSDestination) Decode(buf []byte) error {
```
Comment (vishvananda) [inline route_linux.go]: seems odd to use serialize/decode. I think it should be encode/decode or serialize/deserialize here

File: route.go [hunk 2]
```diff
@@ -25,15 +40,36 @@ type Route struct {
 	Type       int
 	Tos        int
 	Flags      int
+	MPLSDst    int
```
Comment (vishvananda) [inline route.go]: I believe 0 is a valid mpls label id, so it might be necessary to use an *int here?

**Comment by vishvananda** [comment]
thanks for this. A couple of comments inline. Also if you could squash into a single commit that would be great.

**Comment by ishidawataru** [comment]
Reflected your review comments, and squashed into one commit.
PTAL

**Pull request merged**